app/src/ui/diff/side-by-side-diff.tsx

```diff
@@ -114,10 +115,18 @@ interface ISideBySideDiffProps {
   readonly showSideBySideDiff: boolean
 }
 
-interface ISideBySideDiffState {
+interface ISideBySideTextDiffDocument {
   /** The diff that should be rendered */
   readonly diff: ITextDiff
 
+  /** Contents of the old and new files for this diff. */
+  readonly contents: IFileContents | null
```

Comment:
Not super proud of exposing and using `IFileContents` here, but we basically needed everything inside of that interface (except `file`), so… 😕